### DISCUSSION TITLE
Using ADS-B data to calculate emissions

### DISCUSSION THREAD

**Discussion by Juliade7** [junzis/openap#43] [General]
Hello! I am trying to find a way to use my collected ADS-B data to calculate the emissions using the openap module in python. 
Does anyone have experience with this? Any input is very much welcomed as I am quite new to this.
Cheers.

**Comment by 1065305689** [comment]
What kind of data do you collect?

**Comment by ProfHoekstra** [comment]
Historical ADS-B data. From this we distill the performance of aircraft for our open source/ open data models.
